acceptance/analyzer_test.go

```diff
@@ -205,10 +205,16 @@ func testAnalyzerBuilder(apiString string) func(t *testing.T, when spec.G, it sp
 			it("recursively chowns the directory", func() {
 				h.SkipIf(t, runtime.GOOS == "windows", "Not relevant on Windows")
 
+				imgArg := "some-image"
```

Comment:
I prefer that the outside scope will set the variable for the latest API and the if condition will override it for older APIs if needed. Then, if at some point we'll decide to stop supporting some old APIs, we'll be able to just delete the if conditions.

Reply:
643d46d